cli/sub_command.go

```diff
@@ -288,11 +288,7 @@ func (c *subCmd) subscribe(p *fisk.ParseContext) error {
 		}
 
 		c.jsAck = false
-		if c.stream != "" {
-			sub, err = js.Subscribe("", handler, opts...)
-		} else {
-			sub, err = js.Subscribe(c.subject, handler, opts...)
-		}
+		sub, err = js.Subscribe(c.subject, handler, opts...)
```

Comment:
Created a mirror, here is the config

```
$ nats s info foobar --json
{
  "config": {
    "name": "foobar",
    "retention": "limits",
    "max_consumers": -1,
    "max_msgs_per_subject": -1,
    "max_msgs": -1,
    "max_bytes": -1,
    "max_age": 0,
    "max_msg_size": -1,
    "storage": "file",
    "discard": "old",
    "num_replicas": 1,
    "mirror": {
      "name": "foo"
    },
    "sealed": false,
    "deny_delete": false,
    "deny_purge": false,
    "allow_rollup_hdrs": false,
    "allow_direct": false,
    "mirror_direct": false
  },
  "created": "2022-10-04T14:41:10.194868Z",
  "state": {
    "messages": 1000,
    "bytes": 44000,
    "first_seq": 1,
    "first_ts": "2022-10-04T14:40:16.969848Z",
    "last_seq": 1000,
    "last_ts": "2022-10-04T14:40:17.03154Z",
    "num_subjects": 1,
    "consumer_count": 0
  },
  "mirror": {
    "name": "foo",
    "lag": 0,
    "active": 20694250
  }
}
```

Since the subject will be zero value it just gets passed in as "" to this call anyway

```
$ nats sub --stream foobar --count 5
[#1] Received JetStream message: stream: foobar seq 1 / subject: foo / time: 2022-10-04T07:40:16-07:00
hello hello

[#2] Received JetStream message: stream: foobar seq 2 / subject: foo / time: 2022-10-04T07:40:16-07:00
hello hello

[#3] Received JetStream message: stream: foobar seq 3 / subject: foo / time: 2022-10-04T07:40:16-07:00
hello hello

[#4] Received JetStream message: stream: foobar seq 4 / subject: foo / time: 2022-10-04T07:40:16-07:00
hello hello

[#5] Received JetStream message: stream: foobar seq 5 / subject: foo / time: 2022-10-04T07:40:16-07:00
hello hello
```

Reply:
sweet, thanks for checking!